python/cuml/dask/ensemble/randomforestclassifier.py

```diff
@@ -16,14 +16,15 @@
 
 import cudf
 
-from cuml.dask.common import extract_ddf_partitions, \
-    raise_exception_from_futures, workers_to_parts
+from cuml.dask.common import workers_to_parts, \
+    raise_exception_from_futures
```

Comment:
We have been getting in the habit of separating imports on their own lines. Might be a good opportunity to do that here.